content/publishing/firebase-app-distribution.md

```diff
@@ -0,0 +1,35 @@
+---
+description: Deploy a Flutter app to Firebase App Distribution using the Flutter workflow editor
+title: Firebase App Distribution
+weight: 3
+---
+
+Set up publishing to [Firebase App Distribution](https://firebase.google.com/docs/app-distribution) to distribute your Android and iOS apps to testers.
+
+{{<notebox>}}
+This guide only applies to workflows configured with the **Flutter workflow editor**. If your workflow is configured with **codemagic.yaml**, please go to [Publishing an app to Firebase App Distribution](../publishing-yaml/distribution/#publishing-an-app-to-firebase-app-distribution).
+{{</notebox>}}
+
+## Requirements
+
+* [Add Firebase to your Flutter project](https://firebase.google.com/docs/flutter/setup?platform=ios)
```

Comment:
is linking to ios intentional?

Reply:
no, but the page seems to take iOS as the argument by default, I'll update